python/strawReco.py

```
@@ -0,0 +1,225 @@
+    stepper = acts.EigenStepper(fieldMap)
+    propagator = acts.Propagator(stepper, navigator)
+
+    geo_ctx = acts.GeometryContext()
+    mag_ctx = acts.MagneticFieldContext()
+
+    distMin = 99999
+    hitID = -1
+
+    veto_results = []
+
+    for t_idx, track in enumerate(output_tracks):
```

Ruff (0.15.20)

[warning] Loop control variable `t_idx` not used within loop body

Rename unused `t_idx` to `_t_idx`

(B007)